plutus-benchmark/bls12-381-costs/test/bls12-381-costs.golden

```diff
@@ -0,0 +1,102 @@
+Hash n bytestrings onto G1 and add points
+
+    n     Script size             CPU usage               Memory usage
+  ----------------------------------------------------------------------
+    0      77   (0.5%)             100   (0.0%)             100   (0.0%) 
+   10     187   (1.1%)       681707800   (6.8%)           39342   (0.3%) 
+   20     297   (1.8%)      1363794920  (13.6%)           75702   (0.5%) 
+   30     407   (2.5%)      2045882040  (20.5%)          112062   (0.8%) 
+   40     517   (3.2%)      2727969160  (27.3%)          148422   (1.1%) 
+   50     627   (3.8%)      3410056280  (34.1%)          184782   (1.3%) 
+   60     737   (4.5%)      4092143400  (40.9%)          221142   (1.6%) 
+   70     847   (5.2%)      4774230520  (47.7%)          257502   (1.8%) 
+   80     957   (5.8%)      5456317640  (54.6%)          293862   (2.1%) 
+   90    1067   (6.5%)      6138404760  (61.4%)          330222   (2.4%) 
+  100    1177   (7.2%)      6820491880  (68.2%)          366582   (2.6%) 
+  110    1287   (7.9%)      7502579000  (75.0%)          402942   (2.9%) 
+  120    1397   (8.5%)      8184666120  (81.8%)          439302   (3.1%) 
+  130    1507   (9.2%)      8866753240  (88.7%)          475662   (3.4%) 
+  140    1617   (9.9%)      9548840360  (95.5%)          512022   (3.7%) 
+  150    1727  (10.5%)     10230927480 (102.3%)          548382   (3.9%) 
```

Comment:
102.3% of what?

Reply:
102.3% of the maximum CPU ExUnits, as specified in `PlutusBenchmark.ProtocolParameters`, which it appears that I forgot to add to git.  The question is why this PR is passing the CI checks given that that file is imported into `PlutusBenchmarks.Common` [here](https://github.com/input-output-hk/plutus/blob/a9797ed084df6578ffb2d3e61ca1f030ddc1beb7/plutus-benchmark/common/PlutusBenchmark/Common.hs#L24) and its contents are used [here](https://github.com/input-output-hk/plutus/blob/a9797ed084df6578ffb2d3e61ca1f030ddc1beb7/plutus-benchmark/common/PlutusBenchmark/Common.hs#L175).  I'll leave the file out of the repository until we work out what's going on, but it contains this:
```
{-# LANGUAGE NumericUnderscores #-}

-- | 
module PlutusBenchmark.ProtocolParameters (max_tx_size, max_tx_ex_steps, max_tx_ex_mem)
where
  
-- Protocol parameters (June 2023)

-- | This is the "maximum transaction size".  We're just comparing the size of
-- the script with this, so our results may be a little optimistic if the
-- transaction includes other stuff (I'm not sure exactly what "maximum
-- transaction size" means).
max_tx_size :: Integer
max_tx_size = 16384

max_tx_ex_steps :: Integer
max_tx_ex_steps = 10_000_000_000

max_tx_ex_mem :: Integer
max_tx_ex_mem = 14_000_000
```

Reply:
If I check out this branch in a separate copy of the `plutus` repository and try to  build `plutus-benchmark` it does what I'd expect:
```
$ cabal build plutus-benchmark/
Build profile: -w ghc-9.2.7 -O1
In order, the following will be built (use -v for more details):
 - plutus-benchmark-0.1.0.0 (lib:plutus-benchmark-common) (first run)
 [...]
Preprocessing library 'plutus-benchmark-common' for plutus-benchmark-0.1.0.0..
Error: cabal: can't find source for PlutusBenchmark/ProtocolParameters in
common, [...]

Error: cabal: Failed to build lib:plutus-benchmark-common from
plutus-benchmark-0.1.0.0 (which is required by bench:validation-full from
plutus-benchmark-0.1.0.0, bench:validation-decode from
plutus-benchmark-0.1.0.0 and others).
```

Reply:
Weird. Ask Lorenzo?